packages/cxl-lumo-styles/scss/_mixins.scss

```diff
@@ -27,6 +27,10 @@
   &::-webkit-scrollbar-thumb:horizontal {
     min-width: var(--scrollbar-minlength);
   }
+
+  // Firefox
```

Comment:
Is there an ESLint rule to force comment punctuation? Ever since PHPCS does it, I find myself agreeing that without punctuation, it's unclear whether the comment is incomplete.

Reply:
Prettier auto-fix might be even more fun, but not sure if it's safe to auto-punctuate.. although everything in code should mostly end in . ? !, where . ? are 99%.

Reply:
I'm not seeing anything for ESLint or Prettier.